boards/opentitan/earlgrey-cw310/Makefile

```diff
@@ -6,6 +6,8 @@ PLATFORM=earlgrey-cw310
 FLASHID=--dev-id="0403:6010"
 RISC_PREFIX ?= riscv64-elf
 QEMU ?= ../../../tools/qemu/build/qemu-system-riscv32
+# Latest supported commmit of OpenTitan
+OPENTITAN_SUPPORTED_SHA := 217a0168ba118503c166a9587819e3811eeb0c0c
```

Comment:
As per: https://github.com/tock/tock/blob/13bbb77161e365a5bf8c6c41ecc34d4489854f2a/boards/opentitan/README.md?plain=1#L23

Reply:
It's a pain having two copies of this SHA to update. Maybe we could change the README to point to this?

Reply:
Good idea! Updated the `README` to point here.